image_writers_guide/sti.adoc

```diff
@@ -10,70 +10,52 @@
 toc::[]
 
 == Overview
+The main advantage of using STI for building reproducible Docker image is the ease of use for developers. As a builder image author, you must be aware of the two basic requirements for the best possible STI performance: the required image contents and STI scripts.
```

Comment:
[Suggestion] 
The first sentence should either be all singular or all plural. It's incorrect the way it's written now. Probably this: 
s/image/images